extensions/smallrye-health/runtime/src/main/java/io/quarkus/smallrye/health/runtime/SmallRyeHealthRuntimeConfig.java

```diff
@@ -24,6 +24,14 @@ public class SmallRyeHealthRuntimeConfig {
     @ConfigDocMapKey("property-name")
     Map<String, String> additionalProperties;
 
+    /**
+     * Specify the reported DOWN responses should be aligned with RFC 9457 - Problem Details for HTTP APIs.
+     *
+     * @see <a href="https://tools.ietf.org/html/rfc7807">RFC 7807</a>
+     */
+    @ConfigItem(name = "problem-details", defaultValue = "false")
+    boolean problemDetails;
```

Comment:
Question: Should this be a runtime config or can we get away with it being build time?